indispensable/src/commonMain/kotlin/at/asitplus/signum/indispensable/asn1/Asn1Elements.kt

```diff
@@ -127,6 +127,62 @@ sealed class Asn1Element(
         return result
     }
 
+    /**
+     * Convenience function to cast this element to an [Asn1Primitive]
+     * @throws ClassCastException if this element is not a primitive
```

Comment:
it appears from the code that this actually does not throw CCE

Reply:
fixed